vulnerabilities/models.py

```diff
@@ -87,39 +90,38 @@ class Meta:
 
 class ResolvedPackage(models.Model):
     """
-    Relates a vulnerability to package(s) that contain
-    a fix or resolution of this vulnerability.
+    Relates a vulnerability to package(s) that contain a fix or resolution of this vulnerability.
     """
     vulnerability = models.ForeignKey(Vulnerability, on_delete=models.CASCADE)
     package = models.ForeignKey(Package, on_delete=models.CASCADE)
 
 
-class PackageReference(models.Model):
+class Importer(models.Model):
     """
-    One or more identifiers and references for a software package
-    in a package repository, such as a Debian, Maven or NPM repository.
+    Metadata and pointer to the implementation for a source of vulnerability data (aka security advisories)
     """
-    package = models.ForeignKey(Package, on_delete=models.CASCADE)
-    repository = models.CharField(
+    name = models.CharField(max_length=100, unique=True, help_text='Name of the importer')
+    license = models.CharField(max_length=100, blank=True, help_text='License of the vulnerability data')
+    last_run = models.DateTimeField(null=True, help_text='UTC Timestamp of the last run')
+
+    data_source = models.CharField(
         max_length=100,
-        help_text='Repository URL eg:http://central.maven.org',
-        blank=True,
+        help_text='Class name of the data source implementation importable from vulnerabilities.importers',
```

Comment:
Just for clarity, so eventually  our current  `importers`(rust,npm etc ) will be inheriting from the `DataSource` class or `GitDataSource` (for git-based importers)?

Reply:
yes